src/shared/sentry/sentryadapter.h

```diff
@@ -123,6 +123,13 @@ class SentryAdapter final : public QObject {
    */
   Q_INVOKABLE void declineCrashReporting();
 
+  /**
+   * @brief Helper Function, will print the current
+   * stacktrace of the QJSEngine.
+   * @param description - Description of the Stacktrace.
+   */
+  Q_INVOKABLE static void captureQMLStacktrace(const char* description);
```

Comment:
Why `Q_INVOKABLE`?

Reply:
This function will put the qml stack into the Sentry Breadcrumbs. This might be useful if you encounter a qml error state that  you might want to report then using `SentryAdapter::report()`